code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -196,7 +196,7 @@
 	if(user.transferItemToLoc(I, src))
 		to_chat(user, span_notice("You add [I] to [src]."))
 		holdingitems[I] = TRUE
-		return FALSE
+		return TRUE
```

Comment:
is this a bug fix?

Reply:
pretty much. Has to return true so that the attack chain is finished and afterattack isn't called.

Reply:
aight